node-js-basics-101/node_modules/after/test/after-test.js

```diff
@@ -0,0 +1,120 @@
+/*global suite, test*/
+
+var assert = require("assert")
```

Comment:
__[jshint]__
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->